### ISSUE TITLE
Assess clustering

### ISSUE THREAD

**Issue by dkoslicki** [KoslickiLab/L2-UniFrac#7]
Use a clustering assessment metric that takes into consideration ground truth labels (eg. sklearn.metrics.adjusted_rand_score and sklearn.metrics.adjusted_mutual_info_score). Use the true metadata categories as the `labels_true`. For each of the clusters, take a majority vote based on true metadata labels and have this be the `labels_pred` for each cluster. Eg. Say two clusters of six items {{a,b,a},{b,b,b}}. The true labels are {{a,b,a},{b,b,b}} while the predict labels are {{a,a,a},{b,b,b}}. Save clustering metric values and assess how well L1 vs L2 does in clustering by body-site.
